### PR TITLE
Feature/zx3n e hk5 improve jira issue inte testttt

### PR DESCRIPTION
**Pull request by ppezaris**
this is the description x X2

### CODE REVIEW

**Comment by ppezaris** [comment]
test2

**Comment by ppezaris** [comment]
setasdf

**Comment by ppezaris** [comment]
test

File: shared/agent/src/managers/scmManager.ts [hunk 1]
```diff
@@ -201,10 +201,12 @@ export class ScmManager {
 
 		try {
 			repoPath = (await git.getRepoRoot(uri.fsPath)) || "";
-			if (repoPath !== undefined) {
+			if (repoPath.length) {
 				result = await git.getBranches(repoPath);
```
Comment (ppezaris) [inline shared/agent/src/managers/scmManager.ts:205]: here's a comment

File: shared/agent/src/providers/jira.ts [hunk 2]
```diff
@@ -55,12 +56,40 @@ interface CardSearchResponse {
...
 
+export const makeCardFromJira = (card: any, webUrl: string, parentId?: string) => {
+	const { fields = {} } = card;
+	return {
+		id: card.id,
+		url: `${webUrl}/browse/${card.key}`,
+		title: `${card.key} ${fields.summary}`,
+		modifiedAt: new Date(fields.updated).getTime(),
+		tokenId: card.key,
+		body: fields.description,
```
Comment (ppezaris) [inline shared/agent/src/providers/jira.ts:82]: this is part of a review

File: shared/agent/src/providers/jira.ts [hunk 3]
```diff
@@ -216,6 +245,8 @@ export class JiraProvider extends ThirdPartyIssueProviderBase<CSJiraProviderInfo
 								!attributes.hasDefaultValue
 						);
```
Comment (ppezaris) [inline shared/agent/src/providers/jira.ts:247]: this is also a review

File: shared/agent/src/providers/jira.ts (hunk 2 above)
Comment (ppezaris) [reply shared/agent/src/providers/jira.ts:82]: a threaded reply

Comment (ppezaris) [reply shared/agent/src/providers/jira.ts:82]: another threaded reply

**Comment by ppezaris** [comment]
top level comment with markdown
1. foo
2. bar
3. baz

- one
- two
- three

### one

#### two

**Comment by ppezaris** [comment]
&gt; this is a quote<div>*bold*</div><div><br></div>

File: shared/agent/src/managers/scmManager.ts (hunk 1 above)
Comment (ppezaris) [reply shared/agent/src/managers/scmManager.ts:205]: another comment...

Comment (ppezaris) [reply shared/agent/src/managers/scmManager.ts:205]: sodfidsf

Comment (ppezaris) [reply shared/agent/src/managers/scmManager.ts:205]: wtf

Comment (ppezaris) [reply shared/agent/src/managers/scmManager.ts:205]: t r

File: shared/agent/src/managers/scmManager.ts [hunk 4]
```diff
@@ -201,10 +201,12 @@ export class ScmManager {
...
 				const repo = await git.getRepositoryByFilePath(repoPath);
 				repoId = repo ? repo.id || "" : "";
+			} else {
```
Comment (bcanzanella) [inline shared/agent/src/managers/scmManager.ts:208]: test

File: shared/agent/src/providers/jira.ts [hunk 5]
```diff
@@ -30,6 +30,7 @@ interface JiraProject {
 
 interface IssueTypeDescriptor {
 	name: string;
+	iconUrl: string;
```
Comment (bcanzanella) [inline shared/agent/src/providers/jira.ts:33]: asdasda

**Comment by ppezaris** [comment]
is typing here slow? NO!

File: shared/agent/src/providers/jira.ts (hunk 3 above)
Comment (ppezaris) [reply shared/agent/src/providers/jira.ts:247]: this si a reply

Comment (ppezaris) [reply shared/agent/src/providers/jira.ts:247]: &gt; this is also a review and it's edited

replying to it....<br><br>

File: shared/agent/src/managers/scmManager.ts [hunk 6]
```diff
@@ -201,10 +201,12 @@ export class ScmManager {
 
 		try {
 			repoPath = (await git.getRepoRoot(uri.fsPath)) || "";
```
[the remark this answers is not in the document]
Comment (ppezaris) [reply shared/agent/src/managers/scmManager.ts:203]: replying to KKK

File: shared/agent/src/providers/jiraserver.ts [hunk 7]
```diff
@@ -24,6 +24,7 @@ import { CSJiraServerProviderInfo } from "../protocol/api.protocol";
 import { CodeStreamSession } from "../session";
 import { Iterables, log, lspProvider } from "../system";
```
Comment (ppezaris) [inline shared/agent/src/providers/jiraserver.ts:25]: review

Comment (ppezaris) [reply shared/agent/src/providers/jiraserver.ts:25]: review

Comment (ppezaris) [reply shared/agent/src/providers/jiraserver.ts:25]: m

File: shared/agent/src/managers/scmManager.ts (hunk 4 above)
Comment (ppezaris) [inline shared/agent/src/managers/scmManager.ts:208]: jjj

**Pull request closed without merging**